addons/website/models/res_config.py

```diff
@@ -16,37 +16,84 @@ def _default_website(self):
+    google_management_client_secret = fields.Char('Google Client Secret', related='website_id.google_management_client_secret')
 
-    social_twitter = fields.Char(related='website_id.social_twitter')
-    social_facebook = fields.Char(related='website_id.social_facebook')
-    social_github = fields.Char(related='website_id.social_github')
-    social_linkedin = fields.Char(related='website_id.social_linkedin')
-    social_youtube = fields.Char(related='website_id.social_youtube')
-    social_googleplus = fields.Char(related='website_id.social_googleplus')
-    cdn_activated = fields.Boolean('Use a Content Delivery Network (CDN)', related='website_id.cdn_activated')
+    social_twitter = fields.Char("Twitter", related='website_id.social_twitter')
+    social_facebook = fields.Char("Facebook", related='website_id.social_facebook')
+    social_github = fields.Char("GitHub", related='website_id.social_github')
+    social_linkedin = fields.Char("LinkedIn", related='website_id.social_linkedin')
+    social_youtube = fields.Char("Youtube", related='website_id.social_youtube')
+    social_googleplus = fields.Char("Google+", related='website_id.social_googleplus')
+    cdn_activated = fields.Boolean('Content Delivery Network (CDN)', related='website_id.cdn_activated')
     cdn_url = fields.Char(related='website_id.cdn_url')
     cdn_filters = fields.Text(related='website_id.cdn_filters')
-    module_website_form_editor = fields.Selection([
-        (0, 'Use standard forms'),
-        (1, 'Create and customize forms to generate emails, leads, issues and extra information in the checkout process (new snippet available)')
-    ], "Form Builder")
-    module_website_version = fields.Selection([
-        (0, 'No version management and A/B testing (easy)'),
-        (1, 'Allow multiple versions of the same page (advanced)')
-    ], "A/B Testing")
+
+    module_website_form_editor = fields.Boolean("Custom Forms")
+    module_website_version = fields.Boolean("A/B Testing")
+    module_website_twitter = fields.Boolean("Twitter Roller")
+    module_website_blog = fields.Boolean("Blogs")
+    module_website_livechat = fields.Boolean("Live Chat")
+    module_website_forum = fields.Boolean("Forum")
+    module_website_crm = fields.Boolean("Contact Form")
+    module_website_slides = fields.Boolean("Slides")
+    module_website_hr_recruitment = fields.Boolean("Jobs")
+    module_website_sale = fields.Boolean("eCommerce")
+    module_website_contract = fields.Boolean("Subscriptions")
+    module_website_event_sale = fields.Boolean("Event Tickets")
+
     favicon = fields.Binary('Favicon', related='website_id.favicon')
 
     # Set as global config parameter since methods using it are not website-aware. To be changed
     # when multi-website is implemented
     google_maps_api_key = fields.Char(string='Google Maps API Key')
 
+    has_google_analytics = fields.Boolean(
+        "Google Analytics",
```

Comment:
Small note: I think you may concatenate parameter, mabe in two lines. We avoid having long lines, but we do not limitate ourseulf to 80 characters. Moreover here the field definition takes too much vertical space.